[2014-03-28 23:33:13] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in D:\xampp\htdocs\laravel\app\storage\views\7b0156ebe5073385586199ff819aed55:468
Stack trace:
#0 D:\xampp\htdocs\laravel\app\storage\views\7b0156ebe5073385586199ff819aed55(468): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', 'D:\xampp\htdocs...', 468, Array)
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\xampp\htdocs...')
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\xampp\htdocs...', Array)
#3 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\xampp\htdocs...', Array)
#4 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#5 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Http\Response.php(70): Illuminate\View\View->render()
#6 D:\xampp\htdocs\laravel\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'show', Object(Illuminate\View\View))
#10 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'show', Array)
#11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('3')
#12 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#13 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#14 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#15 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#16 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#17 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#18 {main} [] []
[2014-03-28 23:33:55] log.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in D:\xampp\htdocs\laravel\app\storage\views\7b0156ebe5073385586199ff819aed55:468
Stack trace:
#0 D:\xampp\htdocs\laravel\app\storage\views\7b0156ebe5073385586199ff819aed55(468): Illuminate\Exception\Handler->handleError(8, 'Trying to get p...', 'D:\xampp\htdocs...', 468, Array)
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\xampp\htdocs...')
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\xampp\htdocs...', Array)
#3 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\xampp\htdocs...', Array)
#4 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#5 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Http\Response.php(70): Illuminate\View\View->render()
#6 D:\xampp\htdocs\laravel\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'show', Object(Illuminate\View\View))
#10 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'show', Array)
#11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('3')
#12 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#13 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#14 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#15 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#16 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#17 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#18 {main} [] []
[2014-03-28 23:34:13] log.ERROR: exception 'ErrorException' with message 'Undefined variable: hinh_thuc_modal' in D:\xampp\htdocs\laravel\app\storage\views\7b0156ebe5073385586199ff819aed55:467
Stack trace:
#0 D:\xampp\htdocs\laravel\app\storage\views\7b0156ebe5073385586199ff819aed55(467): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', 'D:\xampp\htdocs...', 467, Array)
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\xampp\htdocs...')
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\xampp\htdocs...', Array)
#3 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\xampp\htdocs...', Array)
#4 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#5 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Http\Response.php(70): Illuminate\View\View->render()
#6 D:\xampp\htdocs\laravel\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'show', Object(Illuminate\View\View))
#10 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'show', Array)
#11 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('3')
#12 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#13 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#14 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#15 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#16 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#17 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#18 {main} [] []
[2014-03-28 23:40:56] log.ERROR: exception 'ReflectionException' with message 'Class Giai_thuongsController does not exist' in D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php:308
Stack trace:
#0 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(308): ReflectionClass->__construct('Giai_thuongsCon...')
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(250): Illuminate\Container\Container->build('Giai_thuongsCon...', Array)
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(480): Illuminate\Container\Container->make('Giai_thuongsCon...', Array)
#3 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(983): Illuminate\Foundation\Application->make('Giai_thuongsCon...')
#4 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#5 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#6 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#11 {main} [] []
[2014-03-28 23:42:46] log.ERROR: exception 'ReflectionException' with message 'Class Giai_thuongsController does not exist' in D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php:308
Stack trace:
#0 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(308): ReflectionClass->__construct('Giai_thuongsCon...')
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(250): Illuminate\Container\Container->build('Giai_thuongsCon...', Array)
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(480): Illuminate\Container\Container->make('Giai_thuongsCon...', Array)
#3 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(983): Illuminate\Foundation\Application->make('Giai_thuongsCon...')
#4 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#5 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#6 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#11 {main} [] []
[2014-03-28 23:43:22] log.ERROR: exception 'ReflectionException' with message 'Class Giai_thuongsController does not exist' in D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php:308
Stack trace:
#0 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(308): ReflectionClass->__construct('Giai_thuongsCon...')
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(250): Illuminate\Container\Container->build('Giai_thuongsCon...', Array)
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(480): Illuminate\Container\Container->make('Giai_thuongsCon...', Array)
#3 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(983): Illuminate\Foundation\Application->make('Giai_thuongsCon...')
#4 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#5 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#6 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#11 {main} [] []
[2014-03-28 23:43:34] log.ERROR: exception 'ReflectionException' with message 'Class Giai_thuongsController does not exist' in D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php:308
Stack trace:
#0 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(308): ReflectionClass->__construct('Giai_thuongsCon...')
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(250): Illuminate\Container\Container->build('Giai_thuongsCon...', Array)
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(480): Illuminate\Container\Container->make('Giai_thuongsCon...', Array)
#3 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(983): Illuminate\Foundation\Application->make('Giai_thuongsCon...')
#4 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#5 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#6 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#11 {main} [] []
[2014-03-28 23:43:40] log.ERROR: exception 'ReflectionException' with message 'Class Giai_thuongsController does not exist' in D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php:308
Stack trace:
#0 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(308): ReflectionClass->__construct('Giai_thuongsCon...')
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(250): Illuminate\Container\Container->build('Giai_thuongsCon...', Array)
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(480): Illuminate\Container\Container->make('Giai_thuongsCon...', Array)
#3 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(983): Illuminate\Foundation\Application->make('Giai_thuongsCon...')
#4 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#5 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#6 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#11 {main} [] []
[2014-03-28 23:45:01] log.ERROR: exception 'ReflectionException' with message 'Class Giai_thuongsController does not exist' in D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php:308
Stack trace:
#0 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(308): ReflectionClass->__construct('Giai_thuongsCon...')
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(250): Illuminate\Container\Container->build('Giai_thuongsCon...', Array)
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(480): Illuminate\Container\Container->make('Giai_thuongsCon...', Array)
#3 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(983): Illuminate\Foundation\Application->make('Giai_thuongsCon...')
#4 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#5 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#6 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#11 {main} [] []
[2014-03-28 23:45:20] log.ERROR: exception 'ReflectionException' with message 'Class Giai_thuongsController does not exist' in D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php:308
Stack trace:
#0 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(308): ReflectionClass->__construct('Giai_thuongsCon...')
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(250): Illuminate\Container\Container->build('Giai_thuongsCon...', Array)
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(480): Illuminate\Container\Container->make('Giai_thuongsCon...', Array)
#3 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(983): Illuminate\Foundation\Application->make('Giai_thuongsCon...')
#4 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#5 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#6 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#11 {main} [] []
[2014-03-28 23:45:34] log.ERROR: exception 'ReflectionException' with message 'Class Giai_thuongsController does not exist' in D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php:308
Stack trace:
#0 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(308): ReflectionClass->__construct('Giai_thuongsCon...')
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(250): Illuminate\Container\Container->build('Giai_thuongsCon...', Array)
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(480): Illuminate\Container\Container->make('Giai_thuongsCon...', Array)
#3 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(983): Illuminate\Foundation\Application->make('Giai_thuongsCon...')
#4 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#5 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#6 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#11 {main} [] []
[2014-03-28 23:47:17] log.ERROR: exception 'ReflectionException' with message 'Class Giai_thuongsController does not exist' in D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php:308
Stack trace:
#0 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(308): ReflectionClass->__construct('Giai_thuongsCon...')
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(250): Illuminate\Container\Container->build('Giai_thuongsCon...', Array)
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(480): Illuminate\Container\Container->make('Giai_thuongsCon...', Array)
#3 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(983): Illuminate\Foundation\Application->make('Giai_thuongsCon...')
#4 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#5 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#6 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#11 {main} [] []
[2014-03-28 23:47:28] log.ERROR: exception 'ReflectionException' with message 'Class Giai_thuongsController does not exist' in D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php:308
Stack trace:
#0 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(308): ReflectionClass->__construct('Giai_thuongsCon...')
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(250): Illuminate\Container\Container->build('Giai_thuongsCon...', Array)
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(480): Illuminate\Container\Container->make('Giai_thuongsCon...', Array)
#3 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(983): Illuminate\Foundation\Application->make('Giai_thuongsCon...')
#4 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#5 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#6 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#11 {main} [] []
[2014-03-28 23:47:39] log.ERROR: exception 'ReflectionException' with message 'Class Giai_thuongsController does not exist' in D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php:308
Stack trace:
#0 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(308): ReflectionClass->__construct('Giai_thuongsCon...')
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(250): Illuminate\Container\Container->build('Giai_thuongsCon...', Array)
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(480): Illuminate\Container\Container->make('Giai_thuongsCon...', Array)
#3 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(983): Illuminate\Foundation\Application->make('Giai_thuongsCon...')
#4 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#5 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#6 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#11 {main} [] []
[2014-03-28 23:49:38] log.ERROR: exception 'ReflectionException' with message 'Class Giai_thuongsController does not exist' in D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php:308
Stack trace:
#0 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(308): ReflectionClass->__construct('Giai_thuongsCon...')
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(250): Illuminate\Container\Container->build('Giai_thuongsCon...', Array)
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(480): Illuminate\Container\Container->make('Giai_thuongsCon...', Array)
#3 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(983): Illuminate\Foundation\Application->make('Giai_thuongsCon...')
#4 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#5 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#6 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#11 {main} [] []
[2014-03-28 23:50:09] log.ERROR: exception 'ReflectionException' with message 'Class Giai_thuongsController does not exist' in D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php:308
Stack trace:
#0 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(308): ReflectionClass->__construct('Giai_thuongsCon...')
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(250): Illuminate\Container\Container->build('Giai_thuongsCon...', Array)
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(480): Illuminate\Container\Container->make('Giai_thuongsCon...', Array)
#3 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(983): Illuminate\Foundation\Application->make('Giai_thuongsCon...')
#4 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#5 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#6 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#11 {main} [] []
[2014-03-28 23:50:49] log.ERROR: exception 'ReflectionException' with message 'Class Giai_thuong does not exist' in D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php:352
Stack trace:
#0 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(352): ReflectionParameter->getClass()
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(335): Illuminate\Container\Container->getDependencies(Array)
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(250): Illuminate\Container\Container->build('GiaithuongsCont...', Array)
#3 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(480): Illuminate\Container\Container->make('GiaithuongsCont...', Array)
#4 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(983): Illuminate\Foundation\Application->make('GiaithuongsCont...')
#5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#6 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#11 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#12 {main} [] []
[2014-03-28 23:51:10] log.ERROR: exception 'ReflectionException' with message 'Class Giai_thuong does not exist' in D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php:352
Stack trace:
#0 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(352): ReflectionParameter->getClass()
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(335): Illuminate\Container\Container->getDependencies(Array)
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(250): Illuminate\Container\Container->build('GiaithuongsCont...', Array)
#3 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(480): Illuminate\Container\Container->make('GiaithuongsCont...', Array)
#4 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(983): Illuminate\Foundation\Application->make('GiaithuongsCont...')
#5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#6 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#11 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#12 {main} [] []
[2014-03-28 23:51:33] log.ERROR: exception 'ReflectionException' with message 'Class Giai_thuong does not exist' in D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php:352
Stack trace:
#0 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(352): ReflectionParameter->getClass()
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(335): Illuminate\Container\Container->getDependencies(Array)
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(250): Illuminate\Container\Container->build('GiaithuongsCont...', Array)
#3 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(480): Illuminate\Container\Container->make('GiaithuongsCont...', Array)
#4 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(983): Illuminate\Foundation\Application->make('GiaithuongsCont...')
#5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#6 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#11 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#12 {main} [] []
[2014-03-28 23:51:57] log.ERROR: exception 'ReflectionException' with message 'Class Giai_thuong does not exist' in D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php:352
Stack trace:
#0 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(352): ReflectionParameter->getClass()
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(335): Illuminate\Container\Container->getDependencies(Array)
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(250): Illuminate\Container\Container->build('GiaithuongsCont...', Array)
#3 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(480): Illuminate\Container\Container->make('GiaithuongsCont...', Array)
#4 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(983): Illuminate\Foundation\Application->make('GiaithuongsCont...')
#5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#6 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#11 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#12 {main} [] []
[2014-03-28 23:52:18] log.ERROR: exception 'ReflectionException' with message 'Class Giai_thuong does not exist' in D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php:352
Stack trace:
#0 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(352): ReflectionParameter->getClass()
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(335): Illuminate\Container\Container->getDependencies(Array)
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(250): Illuminate\Container\Container->build('GiaithuongsCont...', Array)
#3 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(480): Illuminate\Container\Container->make('GiaithuongsCont...', Array)
#4 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(983): Illuminate\Foundation\Application->make('GiaithuongsCont...')
#5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#6 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#11 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#12 {main} [] []
[2014-03-28 23:52:29] log.ERROR: exception 'ReflectionException' with message 'Class Giai_thuong does not exist' in D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php:352
Stack trace:
#0 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(352): ReflectionParameter->getClass()
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(335): Illuminate\Container\Container->getDependencies(Array)
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(250): Illuminate\Container\Container->build('GiaithuongsCont...', Array)
#3 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(480): Illuminate\Container\Container->make('GiaithuongsCont...', Array)
#4 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(983): Illuminate\Foundation\Application->make('GiaithuongsCont...')
#5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}('index')
#6 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#11 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#12 {main} [] []
[2014-03-28 23:52:43] log.ERROR: exception 'ReflectionException' with message 'Class Giai_thuong does not exist' in D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php:352
Stack trace:
#0 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(352): ReflectionParameter->getClass()
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(335): Illuminate\Container\Container->getDependencies(Array)
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(250): Illuminate\Container\Container->build('GiaithuongsCont...', Array)
#3 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(480): Illuminate\Container\Container->make('GiaithuongsCont...', Array)
#4 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(983): Illuminate\Foundation\Application->make('GiaithuongsCont...')
#5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#6 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#11 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#12 {main} [] []
[2014-03-28 23:53:59] log.ERROR: exception 'ReflectionException' with message 'Class Giai_thuong does not exist' in D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php:352
Stack trace:
#0 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(352): ReflectionParameter->getClass()
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(335): Illuminate\Container\Container->getDependencies(Array)
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(250): Illuminate\Container\Container->build('GiaithuongsCont...', Array)
#3 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(480): Illuminate\Container\Container->make('GiaithuongsCont...', Array)
#4 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(983): Illuminate\Foundation\Application->make('GiaithuongsCont...')
#5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#6 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#11 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#12 {main} [] []
[2014-03-28 23:56:51] log.ERROR: exception 'ReflectionException' with message 'Class Giai_thuong does not exist' in D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php:352
Stack trace:
#0 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(352): ReflectionParameter->getClass()
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(335): Illuminate\Container\Container->getDependencies(Array)
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Container\Container.php(250): Illuminate\Container\Container->build('Quantrigiaithuo...', Array)
#3 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(480): Illuminate\Container\Container->make('Quantrigiaithuo...', Array)
#4 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(983): Illuminate\Foundation\Application->make('Quantrigiaithuo...')
#5 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#6 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#9 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#10 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#11 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#12 {main} [] []
[2014-03-28 23:57:09] log.ERROR: exception 'InvalidArgumentException' with message 'View [giaithuong.index] not found.' in D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:135
Stack trace:
#0 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(76): Illuminate\View\FileViewFinder->findInPaths('giaithuong.inde...', Array)
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\View\Environment.php(115): Illuminate\View\FileViewFinder->find('giaithuong.inde...')
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(206): Illuminate\View\Environment->make('giaithuong.inde...')
#3 D:\xampp\htdocs\laravel\app\controllers\QuantrigiaithuongController.php(14): Illuminate\Support\Facades\Facade::__callStatic('make', Array)
#4 D:\xampp\htdocs\laravel\app\controllers\QuantrigiaithuongController.php(14): Illuminate\Support\Facades\View::make('giaithuong.inde...')
#5 [internal function]: QuantrigiaithuongController->index()
#6 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('index', Array)
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'index', Array)
#9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#10 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#11 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#12 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#13 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#14 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#15 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#16 {main} [] []
[2014-03-28 23:57:44] log.ERROR: exception 'InvalidArgumentException' with message 'View [giaithuong.index] not found.' in D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php:135
Stack trace:
#0 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\View\FileViewFinder.php(76): Illuminate\View\FileViewFinder->findInPaths('giaithuong.inde...', Array)
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\View\Environment.php(115): Illuminate\View\FileViewFinder->find('giaithuong.inde...')
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Support\Facades\Facade.php(206): Illuminate\View\Environment->make('giaithuong.inde...')
#3 D:\xampp\htdocs\laravel\app\controllers\QuantrigiaithuongController.php(14): Illuminate\Support\Facades\Facade::__callStatic('make', Array)
#4 D:\xampp\htdocs\laravel\app\controllers\QuantrigiaithuongController.php(14): Illuminate\Support\Facades\View::make('giaithuong.inde...')
#5 [internal function]: QuantrigiaithuongController->index()
#6 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(138): call_user_func_array(Array, Array)
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(115): Illuminate\Routing\Controllers\Controller->callMethod('index', Array)
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'index', Array)
#9 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#10 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#11 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#12 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#13 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#14 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#15 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#16 {main} [] []
[2014-03-28 23:58:01] log.ERROR: exception 'Symfony\Component\Routing\Exception\RouteNotFoundException' with message 'Unable to generate a URL for the named route "giai_thuongs.create" as such route does not exist.' in D:\xampp\htdocs\laravel\vendor\symfony\routing\Symfony\Component\Routing\Generator\UrlGenerator.php:134
Stack trace:
#0 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\UrlGenerator.php(192): Symfony\Component\Routing\Generator\UrlGenerator->generate('giai_thuongs.cr...', Array, true)
#1 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Html\HtmlBuilder.php(184): Illuminate\Routing\UrlGenerator->route('giai_thuongs.cr...', Array)
#2 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Support\helpers.php(602): Illuminate\Html\HtmlBuilder->linkRoute('giai_thuongs.cr...', 'Add new giai_th...', Array, Array)
#3 D:\xampp\htdocs\laravel\app\storage\views\ccd55aae71e74e401f81153156ada733(6): link_to_route('giai_thuongs.cr...', 'Add new giai_th...')
#4 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\View\Engines\PhpEngine.php(38): include('D:\xampp\htdocs...')
#5 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\View\Engines\CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('D:\xampp\htdocs...', Array)
#6 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\View\View.php(100): Illuminate\View\Engines\CompilerEngine->get('D:\xampp\htdocs...', Array)
#7 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\View\View.php(81): Illuminate\View\View->getContents()
#8 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Http\Response.php(70): Illuminate\View\View->render()
#9 D:\xampp\htdocs\laravel\vendor\symfony\http-foundation\Symfony\Component\HttpFoundation\Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#10 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#11 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(156): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#12 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Controllers\Controller.php(126): Illuminate\Routing\Controllers\Controller->processResponse(Object(Illuminate\Routing\Router), 'index', Object(Illuminate\View\View))
#13 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(985): Illuminate\Routing\Controllers\Controller->callAction(Object(Illuminate\Foundation\Application), Object(Illuminate\Routing\Router), 'index', Array)
#14 [internal function]: Illuminate\Routing\Router->Illuminate\Routing\{closure}()
#15 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(80): call_user_func_array(Object(Closure), Array)
#16 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Route.php(47): Illuminate\Routing\Route->callCallable()
#17 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Routing\Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#18 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(576): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#19 D:\xampp\htdocs\laravel\vendor\laravel\framework\src\Illuminate\Foundation\Application.php(552): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#20 D:\xampp\htdocs\laravel\index.php(49): Illuminate\Foundation\Application->run()
#21 {main} [] []
